src/main/java/thePackmaster/cards/quantapack/MagicDart.java

```diff
@@ -0,0 +1,58 @@
+package thePackmaster.cards.quantapack;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.actions.common.DamageAllEnemiesAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.cards.AbstractPackmasterCard;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class MagicDart extends AbstractPackmasterCard {
+    public final static String ID = makeID("MagicDart");
+
+    private static final int DAMAGE = 0;
+
+    public MagicDart() {
+        super(ID, 0, CardType.ATTACK, CardRarity.COMMON, CardTarget.ENEMY);
+        baseDamage = DAMAGE;
+    }
+
+    public void getMyName()
+    {
+        if (!this.upgraded)
+            this.rawDescription = cardStrings.DESCRIPTION;
+        else
+            this.rawDescription = cardStrings.UPGRADE_DESCRIPTION;
+    }
+
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        this.baseDamage = AbstractDungeon.player.hand.size();
+        this.calculateCardDamage(m);
```

Comment:
What is happening here? baseDamage should be modified in applyPowers and calculateCardDamage methods, not what you're doing here.

Also, once you've fixed the upp method issue, you don't need to do this getMyName() stuff anymore.